.github/workflows/bump-versions.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Set up Go
-        uses: actions/setup-go@v4
+        uses: actions/setup-go@v5
```

Comment:
note: I can pin these to a hash as the main opentofu/opentofu repo has it if preferred.